### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.centered)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2337]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Documentation - Microsoft MakeCode](https://makecode.microbit.org/docs)
#### Element path
.centered
#### Snippet
`<a class="ui centered item" href="https://makecode.com/" title="Microsoft MakeCode" target="_blank" rel="noopener">Powered by Microsoft MakeCode</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.98 (foreground color: #4183c4, background color: #ffffff, font size: 11.1pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

bottom of docs

**Comment by abchatra** [comment]
fixed
